docs/code-examples/repository-pattern/02_use_with_factories/ilGeoLocationFactory.php

```diff
@@ -0,0 +1,16 @@
+<?
+
+class ilGeoLocationFactory {
```

Comment:
I think I'm entering an area that you are still working on here, right?

Reply:
Yes. I'm still seeking for a good example. The problem here is: Most of the ideas I have for Factories would need additional database access. But since the Repository owns the Factory, this would mean the Factory owns another Repository or worse, the Factory also knows about or even owns the Repository (circular dependency)...

Reply:
Interesting =)

Lets have a look into this in our call as well.